src/backends/sqlite3/statement.cpp

```diff
@@ -610,3 +610,28 @@ sqlite3_statement_backend::make_vector_use_type_backend()
 {
     return new sqlite3_vector_use_type_backend(*this);
 }
+
+db_type sqlite3_statement_backend::exchange_dbtype_for(db_type type) const
+{
+    // Due to SQLite not really having a type system, any integer type may hold
+    // values that should be way outside of its range.
```

Comment:
I believe there is a typo here:
```suggestion
    // values that could be way outside of its range.
```